components/longSparkArrow/LongSparkArrow.js

```diff
@@ -0,0 +1,35 @@
+import {
+  UnfilledSparkArrow,
+  ArrowText,
+  ArrowContainer,
+  ArrowContainerHover,
+  FilledSparkArrow,
+  ArrowTextHover,
+} from './LongSparkArrow.styles';
+import { icons } from '../../style/icons';
+import React, { useState } from 'react';
+
+export default function LongSparkArrow({ arrowText, onClick }) {
+  const [hover, setHover] = useState(false);
+  return (
+    <div
```

Comment:
It looks like this disappears permanently after the first hover is this the expected behavior?